cypress/support/commands.js

```diff
@@ -79,23 +79,54 @@ Cypress.Commands.add('acceptCookiesDialog', () => {
 Cypress.Commands.add('loginFlow', () => {
   cy.visit('/');
 
-  const keycloakLoginUrls = ['localhost', 'front-end-aggregator'];
+  const keycloakLoginFields = {
+    'localhost': {
+      'username': '#username-verification',
+      'password': '#password',
+      'two-step': false
+
+    },
```

Comment:
Can you include `stage.foo` too please?

```suggestion
    },
      'stage.foo.redhat.com:1337': {
      'username': '#username-verification',
      'password': '#password',
      'two-step': false

    },
```